pkg/generate/app/cmd/newapp_test.go

```diff
@@ -137,6 +176,7 @@ func TestValidate(t *testing.T) {
 			}
 		}
 	}
+	envFileOperation("/tmp/envfile123", "delete")
```

Comment:
Makes sense. Will make the modifications. Anything else, are you ok with helper function(evnFileOperations) around in the same file?